### ISSUE TITLE
Registering multiple subpackages simultaneously

### ISSUE THREAD

**Issue by timholy** [JuliaRegistries/Registrator.jl#298]
We just split SnoopCompile into three sub-packages, with `SnoopCompile` itself now a meta-package that loads all three. I tried registering everything and got an error: https://github.com/timholy/SnoopCompile.jl/commit/f9ea9ea947a17048a9ccc099bbec7573b080fda0#commitcomment-40035326

**Comment by timholy** [comment]
Registering the first two one-by-one seems to have worked. But the third one is blocked because it depends on the first two. The meta-package depends on all 3. If we have to go through the entire 3-day waiting period, it will be 6 days before this process is complete. That seems pointless, can we accelerate it?

**Comment by StefanKarpinski** [comment]
Of course, just ping for merges on the Slack #pkg-registration channel. Also, I dare say you can be trusted with commit bit on General if you want that.

**Comment by timholy** [comment]
Thanks! I later realized I could also submit one PR from LocalRegistry, and that will solve the problem nicely. https://github.com/JuliaRegistries/General/pull/16663, which I'm happy to wait the customary 3 days for. It will require manual merging but I can ping folks.

There are times that I wouldn't mind commit bit on General (thanks for the kind offer!), but I also am not (yet) confident of my expertise in knowing exactly what details need to be in the TOML files, so for now I'll keep going without it. Nice to be reminded about the Slack option, I'd forgotten about that as a means of intervention.
